group_manager/static/group_manager/js/group_manager.js

```diff
@@ -911,14 +911,14 @@ $(function () {
       const categoryName = group.category
       const userCanManage = this.canManageGroup(groupName)
 
-      // TRee
+      // Tree
       const $groupList = $('#group-list')
       const $group = $groupList.find('.group[data-name="' + Yoda.escapeQuotes(groupName) + '"]')
       const $oldGroup = $groupList.find('.active')
 
       // group list handling row activation
       const listgroup = $('#tbl-list-groups tr[user-search-result-group="' + Yoda.escapeQuotes(groupName) + '"]')
-      listgroup.addClass('active').siblings().removeClass('active')
+      listgroup.addClass('table-active').siblings().removeClass('table-active')
```

Comment:
To take advantage of being able to use bootstrap's sass variables to customize the color of an active row, had to switch this from active to table-active https://getbootstrap.com/docs/5.0/content/tables/#active-tables